JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -156,7 +154,12 @@ fun ArticleScreen(
                 onToggleFavorite = onToggleFavorite
             )
         }
-    )
+    ) { innerPadding ->
+        val modifier = Modifier
```

Comment:
Nit: inline the modifier. Also, needs a comment to explain this

Reply:
Done!